exporters/otlp/include/opentelemetry/exporters/otlp/otlp_http_client.h

```diff
@@ -71,20 +76,25 @@ struct OtlpHttpClientOptions
   // Additional HTTP headers
   OtlpHeaders http_headers = GetOtlpDefaultHeaders();
 
+  // Concurrency sessions
+  std::size_t concurrency_sessions = 8;
```

Comment:
nit: Is't it better to use `concurrent_sessions`?

Reply:
Thanks, I'm not native english speaker.I will change the name here.